h/views/client.py

```diff
@@ -132,6 +176,20 @@ def absolute_asset_urls(bundle_name):
     }
 
 
+@view_config(route_name='embed',
+             has_feature_flag='use_client_boot_script',
+             http_cache=60 * 5)
```

Comment:
Actually, no, it's fine. We send `Vary: Cookie` so `C-C: public` should be fine here.